functests/tests.sh

```diff
@@ -28,22 +28,22 @@ function test_build() {
     deploy_minio
     uninstall_minio
     uninstall_directcsi
-    # Check uninstall succeeds even if direct-csi is completely gone.
+    # Check uninstall succeeds even if directpv is completely gone.
     "${DIRECT_CSI_CLIENT}" uninstall --crd --force
 }
 
 function do_upgrade_test() {
     wget --quiet --output-document="kubectl-direct_csi_$1" "https://github.com/minio/directpv/releases/download/v$1/kubectl-direct_csi_$1_linux_amd64"
     chmod a+x "kubectl-direct_csi_$1"
 
-    # unmount all direct-csi mounts of previous installation if any.
-    mount | awk '/direct-csi/ {print $3}' | xargs sudo umount -fl
+    # unmount all directpv mounts of previous installation if any.
+    mount | awk '/directpv/ {print $3}' | xargs sudo umount -fl
```

Comment:
older mounts from versions < 1.5 would still have `/direct-csi/` in the mountpaths